docs/Gemfile

```diff
@@ -1,5 +1,3 @@
 source 'https://rubygems.org'
 
 gem 'github-pages'
-
-gem "webrick", "~> 1.7"

```

Comment:
This gem had to be added because of compatibility issues with Ruby 3. Downgrading the Ruby version allows us to remove it again.

Reply:
If we keep this line, it would be possible to use with Ruby 3. Why not support 2 & 3 if it is easy possible?

Reply:
I would not run it if it's not officially supported. There might be other issues that we're not seeing at the surface.